lib/layouts/image_viewer/image_viewer.dart

```diff
@@ -273,6 +279,10 @@ class _ImageViewerState extends State<ImageViewer> with AutomaticKeepAliveClient
                       loadingBuilder: (BuildContext context, ImageChunkEvent ev) {
                         return loader;
                       },
+                      loadFailedChild: Center(
```

Comment:
Before I merge this, can you share what this widget will look like? Is it consistent with what we have in other areas?